tests/e2e/suites/logs/log_suite_test.go

```diff
@@ -0,0 +1,98 @@
+package logs
+
+import (
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/giantswarm/apptest-framework/pkg/config"
+	"github.com/giantswarm/apptest-framework/pkg/state"
+	"github.com/giantswarm/apptest-framework/pkg/suite"
+
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	appsv1 "k8s.io/api/apps/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	isUpgrade = false
+)
+
+func TestConfig(t *testing.T) {
+	var installNamespace = "kube-system"
+
+	appConfig := config.MustLoad("../../config.yaml")
+	appConfig.AppName = "alloy-logs"
```

Comment:
I don't think you should have this now, right?

Reply:
Makes sense yes